ci_framework/roles/dlrn_promote/tasks/get_hash_from_commit.yaml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Get commit.yaml file
+  ansible.builtin.get_url:
+    url: "{{ commit_url }}/commit.yaml"
+    dest: "{{ cifmw_dlrn_promote_workspace }}/commit.yaml"
+    force: true
+  register: result
+  until: result is success
```

Comment:
This returns always a 200? No 201 or other success codes?

Reply:
Done